.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
- [ ] **Required:** To have a professional setup README for a project that has tests, kindly add the **Testing** section with the instructions on how your code should be run.

Reply:
Hey @brytebee 

Thank you for the review, we have corrected that. have a look one more time.